tfjs-backend-webgpu/src/kernels/reduce_webgpu.ts

```diff
@@ -52,7 +54,8 @@ export class ReduceProgram implements WebGPUProgram {
         this.dispatchLayout, this.outputShape, this.workGroupSize);
 
     this.reduceType = reduceType;
-    this.shaderKey = `reduce_${reduceType}`;
+    this.shaderKey =
+        `reduce_${reduceType}_${outputDtype}_${this.outputShape.length}`;
```

Comment:
`${outputDtype}_${this.outputShape.length}` is already in the common part?

Reply:
Great, thanks jiajia!